docs/_pages/documentation.md

```diff
@@ -980,6 +980,18 @@ With trace:
   }
   ```
 
+  By default, trace is displayed only when assertion fails as part of the message. If the trace is needed for successful assertion you can catch it like this:
```

Comment:
🙃 It's _the trace_
🙃 _successful assertions_

Reply:
Thanks, I passed the test through Grammarly. Applied your first comment, but not the other.